docs/cep/reference/extensions/execution/anonymizer.md

```diff
@@ -0,0 +1,160 @@
+
+## Features
+
+Macrometa supports the following data types:
+
+### Address
+
+    ADDRESS_BUILDINGNUMBER
+    ADDRESS_CITY
+    ADDRESS_COUNTRY
+    ADDRESS_COUNTRYCODE
+    ADDRESS_FULLADDRESS
+    ADDRESS_LATITUDE
+    ADDRESS_LONGITUDE
+    ADDRESS_STATE
+    ADDRESS_STATEABBR
+    ADDRESS_STREETADDRESS
+    ADDRESS_STREETADDRESSNUMBER
+    ADDRESS_STREETNAME
+    ADDRESS_TIMEZONE
+    ADDRESS_ZIPCODE
+    ADDRESS_ZIPCODEBYSTATE
+
+### Name
+
+    NAME_FULLNAME
+    NAME_FIRSTNAME
+    NAME_LASTNAME
+    NAME_TITLE
+    NAME_USERNAME
+
+### ID
+
+    ID_SSN
+
+### Credit Card
+
+    CREDITCARD_CREDITCARDTYPE
+    CREDITCARD_CREDITCARDNUMBER
+    CREDITCARD_CREDITCARDEXPIRY
+
+### Company
+
+    COMPANY_DEPARTMENT
+    COMPANY_PRODUCTNAME
+    COMPANY_PRICE
+    COMPANY_PROMOTIONCODE
+    COMPANY_NAME
+    COMPANY_INDUSTRY
+    COMPANY_URL
+    COMPANY_BIC
+    COMPANY_IBAN
+
+### Demographic
+
+    DEMOGRAPHIC_RACE
+    DEMOGRAPHIC_SEX
+    DEMOGRAPHIC_MARITALSTATUS
+    DEMOGRAPHIC_DEMONYM
+
+### Education
+
+    EDUCATOR_CAMPUS
+    EDUCATOR_COURSE
+    EDUCATOR_SECONDARYSCHOOLS
+    EDUCATOR_UNIVERSITY
+
+### Internet
+
+    INTERNET_DOMAINNAME
+    INTERNET_DOMAINSUFFIX
+    INTERNET_DOMAINWORD
+    INTERNET_EMAILADDRESS
+    INTERNET_IPV4ADDRESS
+    INTERNET_IPV4CIDR
+    INTERNET_IPV6ADDRESS
+    INTERNET_IPV6CIDR
+    INTERNET_MACADDRESS
+    INTERNET_PASSWORD
+    INTERNET_SLUG
+    INTERNET_URL
+    INTERNET_UUID
+
+### Job
+
+    JOB_FIELD
+    JOB_POSITION
+    JOB_SENIORITY
+    JOB_TITLE
+
+### Medical
+
+    MEDICAL_DISEASENAME
+    MEDICAL_HOSPITALNAME
+    MEDICAL_MEDICINENAME
+    MEDICAL_SYMPTOMS
+
+### Phone Number
+
+    PHONENUMBER_CELLPHONE
+    PHONENUMBER_PHONENUMBER
+
+### Text
+    TEXT_CHARACTER
+    TEXT_FIXEDSTRING
+    TEXT_PARAGRAPH
+    TEXT_SENTENCE
+    TEXT_WORD
+
+
+
+## Syntax
+
+Anonymizer uses the following syntax:
+
+```js
+	pii:fake(STRING input.string, String fake.function, BOOL invalidate.cache)
+```
+
+		<invalidate.cache> - true, a different fake data will be generated at each call
+				false, once generated the fake data is cached is used for the next calls
+					  
+					  
+## Query Parameters
+
+
+| Name             | Description                                 | Default Value | Possible Data Types | Optional | Dynamic |
+|------------------|---------------------------------------------|---------------|---------------------|----------|---------|
+| input.string     | The input name to create a fake one         |               | STRING              | No       | Yes     |
+| fake.function    | The key to be added                         |               | STRING              | No       | Yes     |
+| invalidate.cache | An optional clean up cache   flag           | false         | BOOL                | Yes      | Yes     |
+
+
+## Example
+
```

Comment:
Add a sentence or two describing what the example is of. Something like, "This example shows how you might anonymize protected patient data for...<use case"